src/System.Buffers/ref/System.Buffers.cs

```diff
@@ -16,3 +16,111 @@ public abstract class ArrayPool<T>
         public abstract void Return(T[] array, bool clearArray = false);
     }
 }
+
+namespace System.Buffers.Binary
+{
+    [System.CLSCompliantAttribute(false)]
+    public static class SpanBinaryExtensions
+    {
+        public static sbyte Reverse(this sbyte value) { throw null; }
+        public static byte Reverse(this byte value) { throw null; }
+        public static short Reverse(this short value) { throw null; }
+        public static ushort Reverse(this ushort value) { throw null; }
+        public static int Reverse(this int value) { throw null; }
```

Comment:
I don't remember this coming up in earlier review - do we really want extension methods hanging off of the primitive integer types? That seems like it'll confuse people or pollute Intellisense.